templates/corporate/hello.html

```diff
@@ -1,562 +1,402 @@
-                                            <a href="/why-zulip/"><img src="{{ static('images/landing-page/hello/organised.svg') }}" alt="" /></a>
-                                            <p><a href="/why-zulip/">Zulip vs Slack &rarr;</a></p>
-                                        </div>
-                                        <div class="other-resources-section">
-                                            <a href="/features/"><img src="{{ static('images/landing-page/hello/featured.svg') }}" alt="" /></a>
-                                            <p><a href="/features/">See all features &rarr;</a></p>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-
-                    <a class="carousel-control left visibility-control hide"
-                      href="#tour-carousel" data-slide="prev"
-                      aria-label="{{ _('Previous') }}" title="{{ _('Previous') }}">
-                        <i class="fa fa-chevron-left"></i>
-                    </a>
-                    <a class="carousel-control right visibility-control"
-                      href="#tour-carousel" data-slide="next"
-                      aria-label="{{ _('Next') }}" title="{{ _('Next') }}">
-                        <i class="fa fa-chevron-right"></i>
-                    </a>
-                    <ol class="carousel-indicators">
-                        <li data-target="#tour-carousel" data-slide-to="0" class="active"></li>
-                        <li data-target="#tour-carousel" data-slide-to="1"></li>
-                        <li data-target="#tour-carousel" data-slide-to="2"></li>
-                        <li data-target="#tour-carousel" data-slide-to="3"></li>
-                        <li data-target="#tour-carousel" data-slide-to="4"></li>
-                        <li data-target="#tour-carousel" data-slide-to="5"></li>
-                        <li data-target="#tour-carousel" data-slide-to="6"></li>
-                        <li data-target="#tour-carousel" data-slide-to="7"></li>
-                    </ol>
-                </div>
-            </div>
+{% block content %}
+    {% include 'zerver/landing_nav.html' %}
+    <div class="portico-hello-page">
+        <div class='body-bg'>
+            <div class='body-bg__layer'></div>
         </div>
-    </div>
-
-    <div class="apps">
-        <div class="triangle"></div>
-        <div class="left-side">
-            <div class="content">
-                <header>
-                    <h1>Apps for every platform.</h1>
-                </header>
-                <p>
-                    Zulip has modern apps for every major platform,
-                    powered by Electron and React Native.
-                </p>
-                <br />
+        <div class="screen-1">
+            <h1>Organized team&nbsp;chat</h1>
```

Comment:
I'm pretty sure these `&nbsp;`s should all just be spaced; just a quirk of the design tool export.

Reply:
Well, it is not design tool export. I made it with a tool and think it it does a reasonable job. I don't want a situation when those words end. Up on a separate lines:
Organized team....
.... chat

This is not the same as 
Organized...
... team chat

So lets review these case instead of removing them all

Reply:
Well I think the proposal was to keep ones where we think line-wrapping of the phrase is plausible.

I would still prefer to not cut up words in this way when it's not serving a functional purpose, because they make tasks like "search for where this string is in the codebase" hard.